src/Autocomplete.js

```diff
@@ -136,7 +160,10 @@ Autocomplete.propTypes = {
   /**
    * Placeholder for input element
    * */
-  placeholder: PropTypes.string
+  placeholder: PropTypes.string,
+  onChange: PropTypes.func,
+  onAutocomplete: PropTypes.func,
+  value: PropTypes.string
 };
```

Comment:
Please add a description to these fields so that they can be parsed and appear in the documentation, like L160 :)

Reply:
Sure, I will